### PR TITLE
add inequality checkers

### PR DESCRIPTION
**Pull request by AimeeKnight**
Adds #20. I'd like to pull out some of the checkers like we began in the move-checkers branch, but wanted to discuss an approach with you prior.

### CODE REVIEW

File: src/checkers.js [hunk 1]
```diff
@@ -413,6 +415,24 @@ function getCheckers(disabled) {
     }, {type}, disabled);
   }
 
+  function lessThanCheckGetter(min) {
```
Comment (kentcdodds) [inline src/checkers.js]: Codacy Issue found: ['lessThanCheckGetter' is defined but never used.](https://www.codacy.com/app/kentcdodds/api-check/file/738631697/issues/source?bid=2426116&fileBranchId=2432553#l418)

File: src/checkers.js [hunk 2]
```diff
@@ -413,6 +415,24 @@ function getCheckers(disabled) {
...
+    const type = 'Number';
+    return setupChecker(function lessThanChecker(val, name, location) {
+      if (typeof val !== 'number' || val > min) {
+        return getError(name, location, type);
+      }
+    }, {type}, disabled);
+  }
+
+  function greaterThanCheckGetter(max) {
```
Comment (kentcdodds) [inline src/checkers.js]: Codacy Issue found: ['greaterThanCheckGetter' is defined but never used.](https://www.codacy.com/app/kentcdodds/api-check/file/738631697/issues/source?bid=2426116&fileBranchId=2432553#l427)

**Comment by kentcdodds** [comment]
Please ignore codacy comments... I'll look at this asap.

File: src/checkers.js [hunk 3]
```diff
@@ -413,6 +415,24 @@ function getCheckers(disabled) {
     }, {type}, disabled);
   }
 
+  function lessThanCheckGetter(min) {
+    const type = 'Number';
```
Comment (kentcdodds) [inline src/checkers.js]: For us, the `type` is used in the error message displayed to the user. So perhaps a more descriptive type here would be `const type = `lessThan[${min}]`;` or perhaps `const type = `< ${min}`;`. That way the output might read something like: ``value` at `Argument 1` must be `lessThan[3]`` or ``value` at `Argument 1` must be `< 3``

**Comment by kentcdodds** [comment]
Thi is awesome @AimeeKnight! Looking forward to future contributions :-D

File: src/checkers.js (hunk 3 above)
Comment (AimeeKnight) [inline src/checkers.js]: Ok, I see that now. I'll make the change and push it up tomorrow.

**Comment by AimeeKnight** [comment]
This should be similar to the range error message now.

**Comment by kentcdodds** [comment]
Hey Aimee! Looks almost 100% perfect. The last thing that I would like is if you would not commit anything you change in the `dist` directory. I prefer to only commit that for a release because it can make git diffs a little crazy (because it's a generated file).

If you would like, you can squash all of your commits for this PR into a single commit to clean the git history a bit. You can do this the [hard way](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) (more hard-core), or the easy way. I prefer the easy way using `$ git reset --soft HEAD~3`. This will remove all of your commits and leave the delta of all the changes. Then you can make any modifications and commit only what you want as a single commit. This is a history rewrite operation, so I recommend double checking that you're pushing the right stuff (because you'll have to do a force push `$ git push -f`).

If you're curious what the `HEAD~3` thing is all about, see [this](http://stackoverflow.com/questions/1955985/what-does-the-caret-character-mean).

**Comment by AimeeKnight** [comment]
Sorry about that, definitely makes sense. Thank you for the explanation and link.

**Comment by kentcdodds** [comment]
Perfect! I'll release tomorrow :-)

**Pull request merged**